WordPress-iOS-Shared/Core/WPFontManager.m

```diff
@@ -172,7 +172,7 @@ + (UIFont *)merriweatherBoldFontOfSize:(CGFloat)size
 + (void)dynamicallyLoadFontResourceNamed:(NSString *)name
 {
     NSString *resourceName = [NSString stringWithFormat:@"%@/%@", kBundle, name];
-    NSURL *url = [[NSBundle mainBundle] URLForResource:resourceName withExtension:@"ttf"];
+    NSURL *url = [[NSBundle bundleForClass:[WPFontManager class]] URLForResource:resourceName withExtension:@"ttf"];
```

Comment:
Fixed in https://github.com/wordpress-mobile/WordPress-Shared-iOS/commit/c17b0bd12e2e27ea7d4338c734a3eea0d0a94f6e